### PR TITLE
Test

### PR DESCRIPTION
**Pull request by yash244466666**
### UNIT TEST 🧪
**Added unit tests for all the methods in all the classes:**

- Book
- Person
- Student
- Teacher
- Classroom
- Rental

### CODE REVIEW

File: spec/book_spec.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require 'rspec'
```
Comment (mwanawabangona) [inline spec/book_spec.rb]: - [ ] Good job so far! 🥂 
- [ ] if i run rspec spec i'm getting an error see screenshot below
- [ ] You dont need to require rspec if its the same folder.
- [ ] please remove it to solve the error

image

File: spec/rental_spec.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+require './rental'
+require './book'
+require './person'
+require 'rspec'
```
Comment (mwanawabangona) [inline spec/rental_spec.rb]: - [ ] Good job so far! 🥂 
- [ ] if i run rspec spec i'm getting an error see screenshot below
- [ ] You dont need to require rspec if its the same folder.
- [ ] please remove it to solve the error

image

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @team!,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

- [ ]  The unit tests  was done using Rspec. that means when testing them you need to use `rspec spec` and not `rubocop --color` which used under similar test.
- [ ] if you remove the `require rspec` in the requested folders and Run rspec spec the test will pass

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by yash244466666** [comment]
### Added chages as instructed

- [x] Removed the `require 'rspec'`

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @yash244466666 , 
Congratulations!GIF

Great job on implementing the change requested by the previous reviewer :clap: . Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Good commit messages. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**